### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rstockbridge**
I observed that my build would still fail due to errors from Android Lint even if I set `failOnError` to `false` for gnag. To fix this, I needed to specify `abortOnError false` for Android LintOptions. I've updated the README to indicate this.

Please confirm that my Kotlin syntax is correct, or make any necessary changes. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +190,34 @@ gnag {
...
+    
+    </details>
+    
+    <details open>
+    <summary><b>kotlin</b></summary>
+
+    ```
+    android {
+        lintOptions {
+            abortOnError = false
```
Comment (btkelly) [inline README.md]: I'm not sure that it matters but if we want to remain consistent with the rest of the Kotlin documentation we might want to change the syntax to `abortOnError(false)`

File: README.md [hunk 2]
```diff
@@ -190,7 +190,34 @@ gnag {
...
+    android {
+        lintOptions {
+            abortOnError false
+        }
+    }
+    ```
+    
+    </details>
+    
+    <details open>
```
Comment (btkelly) [inline README.md]: Also to remain consistent with the rest of the Kotlin docs remove the `open` option here and have it just be `<details>`

**Review by btkelly — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stkent) [reply README.md]: That depends on how `abortOnError` is defined in the plugin; let me check real quick.

Comment (stkent) [reply README.md]: It's unclear, doing some experiments!

Comment (stkent) [reply README.md]: From #218 can confirm, the syntax in Kotlin build scripts needs to be:

```java
android {
    lintOptions {
        setAbortOnError(false)
    }
}
```

Screen Shot 2019-04-02 at 3 16 32 PM
Screen Shot 2019-04-02 at 3 16 44 PM

**Comment by btkelly** [comment]
Nice 💯

**Pull request merged**